contracts/.prettierignore

```diff
@@ -0,0 +1,3 @@
+cache/
```

Comment:
there's already a prettierignore at the root of the project

Reply:
Gotcha.